test/plugin.test.js

```diff
@@ -1,77 +0,0 @@
-import document from 'global/document';
```

Comment:
was this file removed on purpose?

Reply:
(no - checking)

Reply:
I've put it back - not having a good day it seems

Reply:
haha, no worries. Don't forget to take some time for yourself!